onboarding-scripts/Makefile

```diff
@@ -0,0 +1,55 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=onboarding-scripts
+#PKG_VERSION:=dev
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=1
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=$(TOPDIR)/../../packages/onboarding-scripts
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_VERSION:=$(PKG_VERSION)
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+
+CMAKE_INSTALL:=1
+
+CMAKE_OPTIONS += -DSTAGING_DIR=$(STAGING_DIR)
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+define Package/onboarding-scripts
+  SECTION:=utils
+  CATEGORY:=Utilities
+  TITLE:=Ci40 onboarding-scripts
+  DEPENDS:=+luci +libubox-lua +libcurl +luci-mod-rpc +luci-ssl-openssl +uhttpd-mod-tls +avahi-dbus-daemon +avahi-dnsconfd +letmecreate
+endef
+
+define Package/onboarding-scripts/description
+  Imagination Technologies Ci40 onboarding scripts
+endef
+
+define Build/Compile
+		$(call Build/Compile/Default,all)
+endef
+
+define Build/Clean
+		$(call Build/Compile/Default,clean)
+endef
+
```

Comment:
Do we actually need to define Build/Compile and Build/Clean if you are using the default functions ?